cardano-cli/test/cardano-cli-golden/Test/Golden/Shelley/StakeAddress/KeyHash.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Test.Golden.Shelley.StakeAddress.KeyHash where
+
+import           Test.Cardano.CLI.Util as OP
+
+import           Hedgehog (Property, (===))
+import qualified Hedgehog.Extras.Test.Base as H
+import qualified Hedgehog.Extras.Test.File as H
+
+{- HLINT ignore "Use camelCase" -}
+
+hprop_golden_shelleyStakeAddressKeyHash :: Property
+hprop_golden_shelleyStakeAddressKeyHash = propertyOnce . H.moduleWorkspace "tmp" $ \_ -> do
+  verificationKeyFile <- noteInputFile "test/cardano-cli-golden/files/input/shelley/keys/stake_keys/verification_key"
+  goldenVerificationKeyHashFile <- noteInputFile "test/cardano-cli-golden/files/input/shelley/keys/stake_keys/verification_key.key-hash"
+
+  verificationKeyHash <- execCardanoCLI
+    [ "stake-address","key-hash"
+    , "--stake-verification-key-file", verificationKeyFile
+    ]
+
+  goldenVerificationKeyHash <- H.readFile goldenVerificationKeyHashFile
+
+  verificationKeyHash === goldenVerificationKeyHash
```

Comment:
Golden tests should use on of the golden test functions:

* `diffVsGoldenFile`
* `diffFileVsGoldenFile`